plugins/pa11y/.toolkitrc.yml

```diff
@@ -2,4 +2,4 @@ plugins:
   - '@dotcom-tool-kit/doppler'
 
 hooks:
-  'test:review': Pa11y
+  'test:local': Pa11y
```

Comment:
**issue**: removing the default link between the `test:review` hook and the `Pa11y` task is a breaking change as it will mean Pa11y will stop running in end to end tests. we should keep the association between the two until we release a new major version
```suggestion
  'test:review': Pa11y
  'test:local': Pa11y
```

Reply:
The tasks doesn't do anything in `test:review` so I believe we should release this deprecation as a breaking change.

Reply:
To clarify: do you mean that Pa11y doesn't work in CI? Why do you say the tasks don't do anything in the `test:review` hook?

Reply:
Pa11y fails to execute the tests when running on CircleCI however it fails silently and the job will not fail which is misleading people to believe nothing is wrong.

I have updated the commit message to be a breaking change and changed the wording to be clear of the intent.